compare.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/python
+#
+# Compares the speed of two different compilers on one or more benchmarks in
+# the suite.
+
+from __future__ import division, print_function
+
+import os
+import subprocess
+import sys
+import time
+
+all_dirs = [
+    'futures-rs-test-all',
+    'helloworld',
+    'html5ever-2016-08-25',
+    'hyper.0.5.0',
+    'inflate-0.1.0',
+    'issue-32062-equality-relations-complexity',
+    'issue-32278-big-array-of-strings',
+    'jld-day15-parser',
+    'piston-image-0.10.3',
+    'regex.0.1.30',
+    'rust-encoding-0.3.0',
+    'syntex-0.42.2',
+    'syntex-0.42.2-incr-clean',
```

Comment:
It would be nice if we could collect this information at runtime; maybe even as a list of shell arguments?

I think we already somewhat do that, but I'm not sure that there are benefits to keeping a static list here.